it/controllers/IndexControllerISpec.scala

```diff
@@ -73,6 +81,43 @@ class IndexControllerISpec extends IntegrationSpecCommonBase {
     latePaymentPenalties = Some(Seq.empty[LatePaymentPenalty])
   )
 
+  val getPenaltyDetailsPayloadWithAddedPoint = GetPenaltyDetails(
+    totalisations = None,
+    lateSubmissionPenalty = Some(LateSubmissionPenalty(
+      summary = LSPSummary(
+        activePenaltyPoints = 2,
```

Comment:
Shouldn't this be activePenaltyPoints = 1?

Reply:
Yes, but in the payload there is only one penalty, so it makes sense to change the test (most likely the previous test data was wrong)

Reply:
Fixed the tests